src/ObjectGraph.es6.js

```diff
@@ -248,12 +276,14 @@ ObjectGraph.prototype.visitObject = function(o) {
 
   var dataMap = this.storeObject(id);
   var metadataMap = this.storeMetadata(id);
+  this.visitPrototype(o, dataMap);
 
-  // Enqueue work: Visit o's prototype and property descriptors.
-  this.q.enqueue(this.visitPrototype.bind(this, o, dataMap));
```

Comment:
Put visitPrototype into the queue will result to an 'repeated store proto' error. So I just called visitPrototype function instead of putting into the task queue.